packages/core/admin/utils/get-plugins-path.js

```diff
@@ -1,15 +1,23 @@
 'use strict';
 
-const { join, resolve } = require('path');
+const { join, resolve, sep } = require('path');
 const fs = require('fs-extra');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const glob = require('glob');
 
 // Only for dev environement
 const getPluginsPath = () => {
   const rootPath = resolve(__dirname, '..', join('..', '..', '..', 'packages'));
```

Comment:
```suggestion
  const rootPath = resolve(__dirname, '..', '..', '..', '..', 'packages');
```
I wouldn't use split/join or we could run into potential problems in the future. Would this change solve the problem instead?